### DISCUSSION TITLE
Does Nornir support socks?

### DISCUSSION THREAD

**Discussion by luxxfr** [nornir-automation/nornir#662] [Q&A]
Hi,

Does anyone know whether Nornir supports socks?

**Comment by dbarrosop** [comment]
You will have to check the connection plugin you are interested in and see if it supports socks.

**Comment by ktbyers** [comment]
Netmiko + Nornir plugin should support this using the `sock=` argument:

https://github.com/ktbyers/netmiko/blob/v3.3.3/netmiko/base_connection.py#L90

**Reply by luxxfr** [reply]
thanks
